week1/homework/src/server.js

```diff
@@ -5,13 +5,38 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+
 function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
-  });
+    if (request.url.startsWith('/state')) {
+      response.writeHead(200, { 'Content-Type': 'application/json' });
+      response.write(JSON.stringify({ state: state }));
+    }
+ else if (request.url === '/add') {
```

Comment:
Hi Ramazan - the indentation of the `else-if` blocks aren't correct. Could you please check?

Reply:
I am doing in a correct way but Visual Studio auto correct it like that.